libs/vue/src/builders/library/schema.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$id": "https://json-schema.org/draft-07/schema",
+  "title": "Library builder",
+  "description": "Build library",
+  "type": "object",
+  "properties": {
+    "dest": {
+      "type": "string",
+      "description": "Specify output directory."
+    },
+    "clean": {
+      "type": "boolean",
+      "description": "Remove the dist directory before building the project.",
+      "default": true
+    },
+    "report": {
+      "type": "boolean",
+      "description": "Generate report.html to help analyze bundle content.",
+      "default": false
+    },
+    "reportJson": {
+      "type": "boolean",
+      "description": "Generate report.json to help analyze bundle content.",
+      "default": false
+    },
+    "skipPlugins": {
+      "type": "string",
+      "description": "Comma-separated list of plugin names to skip for this run."
+    },
+    "entry": {
+      "type": "string",
+      "description": "The full path for the main entry point to your library, relative to the current workspace."
+    },
+    "watch": {
+      "type": "boolean",
+      "description": "Watch for changes.",
+      "default": false
+    },
+    "tsConfig": {
+      "type": "string",
+      "description": "The full path for the TypeScript configuration file, relative to the current workspace."
+    },
+    "inlineVue": {
+      "type": "boolean",
+      "description": "Bundle Vue with your library rather than externalizing it.",
```

Comment:
Align this description with Vue CLI's [command line description](https://github.com/vuejs/vue-cli/blob/v4.3.1/packages/%40vue/cli-service/lib/commands/build/index.js#L32).